src/redux/home/homeSlice.js

```diff
@@ -0,0 +1,47 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+export const fetchCities = createAsyncThunk(
+  'cities/fetchCities',
+  async () => {
+    try {
+      const response = await fetch(
+        'https://api.openweathermap.org/data/2.5/weather?lat=25.21490645&lon=55.174787069501136&appid=21aebf13ffa527bf0b5a8674bc387578',
+      );
+      const data = await response.json();
```

Comment:
- [ ]  It would be better also as suggested by the previous reviewer, to kindly ensure there is the mock test for your API. you may check this [video](https://www.youtube.com/watch?v=dL_TsWTASfg) for assistance 👍

Reply:
@TuyishimireEric , please help and guide me for the mock test